Code/Source/svFSI/nn.cpp

```diff
@@ -606,19 +611,39 @@ void gnnb(const ComMod& com_mod, const faceType& lFa, const int e, const int g,
     }
   }
 
-  // Correcting the position vector if mesh is moving
+  // Correcting the geometry if mesh is moving or if we want the 
+  // area-weighted normal in a different configuration
   //
   for (int a = 0; a < eNoN; a++) {
     int Ac = msh.IEN(a,Ec);
     for (int i = 0; i < lX.nrows(); i++) {
+      // Get position vector
       lX(i,a) = com_mod.x(i,Ac);
     }
-
     if (com_mod.mvMsh) {
       for (int i = 0; i < lX.nrows(); i++) {
+        // Add mesh displacement
         lX(i,a) = lX(i,a) + com_mod.Do(i+nsd+1,Ac);
       }
     }
+    else if (cfg == 'r') {
+        // Do nothing
```

Comment:
Probably we do not need at all the else if statement for 'r', given the Do Nothing (a comment would be sufficient)

Reply:
Changed this part of code to switch statement.